daemon/graphdriver/aufs/aufs.go

```diff
@@ -573,7 +573,7 @@ func (a *Driver) aufsMount(ro []string, rw, target, mountLabel string) (err erro
 
 	offset := 54
 	if useDirperm() {
-		offset += len("dirperm1")
+		offset += len(",dirperm1")
```

Comment:
In the original it was `50`. I'm quite sure this was added with a margin of error as it calls through the selinux package and can't precisely predict how many characters would be added if that package should change.